src/ert/_c_wrappers/enkf/site_config.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, user_config_file=None, config_content=None, config_dict=None)
             )
 
             env_var_list.convertToCReference(None)
-            umask = config_dict.get(ConfigKeys.UMASK)
+            umask = config_dict.get(ConfigKeys.UMASK, 0o022)
```

Comment:
what is `0o022` this exactly when I use this value as a default it results in a Resconfig with a SiteConfig with umask with value 18... and when I initialize a simple config from file the default value of the unmak is 2.

Then why not use config_dict.get(ConfigKeys.UMASK, 2) ?

Reply:
Hmm very good issue you're raising - so first of, `0o022` is (python) base 8 int notation, meaning 2*1 + 2*8 = 18.

the inconsistency with initiating a ResConfig from config file and getting a 2 is of course something that should be fixed, I will look into it.

the reason for using octal notation, I guess, is just convention / what some people might be used to in terms of linux file permissions. for those used to that notation, this way of writing it is easy to parse, while an `18` in base 10 notation is more difficult to interpret.